documentation/puppet-api/v3/task_detail.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "Task detail",
```

Comment:
is it normal to align these

Reply:
I have no idea, I mainly just copied what already existed